PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchViewModel.swift

```diff
@@ -103,6 +134,7 @@ class SearchViewModel: ObservableObject {
     }
 
     private func updateRecentSearches(with searchTerm: String) -> [String] {
+        guard isPremium else { return [] }
```

Comment:
had to move this check here for my tests to pass for free users, not sure why but it works. If anyone has insight on why moving this allows `test_updateScope_forFreeUser_withArchiveAndTerm_showsResults` to pass, I'm open ears!

Reply:
Got on a call with @timc-mozilla and @OrlaM during our weekly search meetings. It seems that this may be due to the asynchronous search call via our search service. However, we are using a completion block to mock that call, so something funky seems to be happening. We discussed that our future strategy as a team should either be to consistently use the async / await pattern or completion block pattern. Will open this discussion in slack.